test/helper.rb

```diff
@@ -33,7 +33,7 @@
 Thread.abort_on_exception = true
 
 $debugging_info = []
-$debugging_hold = false   # needed for TestCLI#test_control_clustered
```

Comment:
Seems like the test mentioned here no longer exists.

Reply:
@MSP-Greg ?

Reply:
Comment was added over 5 years ago in https://github.com/puma/puma/commit/4c8d4d6921a1b9b9155ab83b09a632c377401398

Reply:
Looks like https://github.com/puma/puma/commit/9a8fcb28190673aecaa129632b7c640517430ff5 moved & renamed (slightly) the test

Reply:
Looks like we can remove debugging_hold and then update the comment

Reply:
> Looks like we can remove `$debugging_hold` and then update the comment

Yes.  A quick check show it isn't used anymore, I.think.  Also remvoe the comment on line 227.

The mentioned test (test_control_clustered) was starting a clustered server using `Puma::CLI.new`.  It has since been been removed (or moved).